Fix AttributeError in `Lockfile.is_up_to_date()`
